pkg/kudoctl/cmd/upgrade.go

```diff
@@ -111,5 +111,5 @@ func runUpgrade(args []string, options *options, fs afero.Fs, settings *env.Sett
 
 	resources.OperatorVersion.SetNamespace(settings.Namespace)
 
-	return upgrade.OperatorVersion(kc, resources.OperatorVersion, options.InstanceName, options.Parameters, resolver)
+	return upgrade.OperatorVersion(kc, packageToUpgrade, resources.OperatorVersion, options.InstanceName, options.Parameters, resolver)
```

Comment:
Consistency nitpick: Let's change the function signature to expect depencies instead of a `resolver`, similar to your changes to `install.Package` and `install.OperatorAndOperatorVersion`.

Reply:
Good point.